app/src/main/java/com/alorma/compose/settings/example/MainActivity.kt

```diff
@@ -12,13 +12,7 @@ import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
 import com.alorma.compose.settings.example.ui.Navigation
-import com.alorma.compose.settings.example.ui.screens.AppSettingsScreen
-import com.alorma.compose.settings.example.ui.screens.CheckboxesScreen
-import com.alorma.compose.settings.example.ui.screens.ListScreen
-import com.alorma.compose.settings.example.ui.screens.MenuLinksScreen
-import com.alorma.compose.settings.example.ui.screens.SlidersScreen
-import com.alorma.compose.settings.example.ui.screens.SwitchesScreen
-import com.alorma.compose.settings.example.ui.screens.TopLevelScreen
+import com.alorma.compose.settings.example.ui.screens.*
```

Comment:
Could you revert the wildcard imports? I prefer to have a list of imports instead of generic import.

Btw. I should add a codeStyle to the project.

Reply:
Fixed